electron/utils.ts

```diff
@@ -149,6 +163,9 @@ export const getUserInfo = (): UserInfo => {
 }
 
 const updateGame = (game: string) => {
+  if (!isOnline()) {
+    console.log(`App offline, skipping update for game '${game}'.`)
```

Comment:
Shouldn't we return here something to make it stop trying to update anyway?

Reply:
updateGame is often called by a user interaction. Maybe its an idea to show a warning to the user that tell them why there is nothing happening? Like a dialog that tell them they can not update the game because they appear to be offline.

Reply:
> Shouldn't we return here something to make it stop trying to update anyway?

Yeah I forgot to actually return, don't know how that worked...

Reply:
Fixed.